build-tools/packages/build-tools/src/type-test-generator/main.ts

```diff
@@ -62,23 +85,29 @@ const currentFile = new Project({
 
 const previousTsConfigPath = `${previousBasePath}/tsconfig.json`;
 let previousFile: SourceFile;
-if (existsSync(previousTsConfigPath)) {
-	const project = new Project({
-		skipFileDependencyResolution: true,
-		tsConfigFilePath: previousTsConfigPath,
-	});
-	previousFile = project.getSourceFileOrThrow("index.ts");
+const project = new Project({
+	skipFileDependencyResolution: true,
+	tsConfigFilePath: existsSync(previousTsConfigPath) ? previousTsConfigPath : undefined,
+});
+// Check for existence of alpha and add appropriate file
+if (existsSync(typeDefinitionFilePath)) {
+	project.addSourceFilesAtPaths(typeDefinitionFilePath);
+	previousFile = project.getSourceFileOrThrow("<package-name>-alpha.d.ts");
```

Comment:
Presumably we need to use the actual package name here, rather than "<package-name>", unless the tooling is doing something special for us here.

Reply:
And note that it needs to be the name of the _previous_ version of the package, in the event of a package rename.